addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -978,7 +984,7 @@ export class Rtc extends Record {
             }
         }
         const updatedTrack = type === "camera" ? this.state.cameraTrack : this.state.screenTrack;
-        await this.network.updateUpload(type, updatedTrack);
+        await this.network?.updateUpload(type, updatedTrack);
```

Comment:
Looks like this is the only relevant diff in this PR

Reply:
Indeed.